src/test/scala/it/pagopa/interop/backendforfrontend/AuthorizationApiServiceSpec.scala

```diff
@@ -12,33 +11,44 @@ import org.scalatest.wordspec.AnyWordSpecLike
 
 import java.util.UUID
 import scala.concurrent.Future
-import scala.jdk.CollectionConverters.{MapHasAsJava, SeqHasAsJava}
 import scala.util.{Failure, Success}
+import SpecHelper._
 
 class AuthorizationApiServiceSpec extends AnyWordSpecLike with SpecHelper with ScalatestRouteTest {
 
-  "Generating a session token" ignore {
+  "Generating a session token" should {
 
     "succeed" in {
 
-      val uid: String                       = UUID.randomUUID().toString
-      val organization: Map[String, String] = Map("id" -> "id", "role" -> "role", "fiscalCode" -> "fiscalCode")
-      val claimSet: Map[String, AnyRef]     = Map("uid" -> uid, "organization" -> organization.asJava)
+      val uid: String = UUID.randomUUID().toString
 
-      val builder: JWTClaimsSet.Builder = new JWTClaimsSet.Builder()
-      val jwtClaimsSet: JWTClaimsSet    = builder.claim("uid", uid).claim("organization", organization.asJava).build()
+      val jwtClaimsSet = Map(
+        "organization" -> Map("id" -> "id", "fiscalCode" -> "fiscalCode", "roles" -> List(Map("role" -> "admin"))),
```

Comment:
since we're here, may we add more roles to test the comma separated string?

Reply:
How about now?